Example/StripeConnectExample/StripeConnectExample/AppSettingsView.swift

```diff
@@ -8,32 +8,56 @@
 import SwiftUI
 
 struct AppSettingsView: View {
-    
+
     @Environment(\.dismiss) var dismiss
     @Environment(\.viewControllerPresenter) var viewControllerPresenter
-    
+
     // App info is sometimes nil, if for example it fails to load.
     var appInfo: AppInfo?
-    
+
     @State var selectedMerchant: MerchantInfo?
     @State var serverURLString: String = AppSettings.shared.selectedServerBaseURL
-    
+
     var isCustomEndpointValid: Bool {
         guard let url = URL(string: serverURLString) else { return false }
         return UIApplication.shared.canOpenURL(url)
     }
-    
+
+    var isUsingCustomMerchant: Bool {
+        guard let selectedMerchant,
+              let appInfo else {
+            return true
+        }
+        return !appInfo.availableMerchants.contains(selectedMerchant)
+    }
+
+    var isMerchantIdValid: Bool {
+        if !isUsingCustomMerchant {
+            return true
+        } else if let selectedMerchant {
+            return selectedMerchant.id.starts(with: "acct_")
+            && selectedMerchant.id.count > 5
```

Comment:
```suggestion
            return isMerchantIdValid(selectedMerchant.id)
```

Reply:
Oops, I meant to delete `isMerchantIdValid(_ id)` 🤦‍♀️